ouroboros-consensus/src/Ouroboros/Storage/VolatileDB/Util.hs

```diff
@@ -75,32 +78,23 @@ wrapFsError :: Monad m
 wrapFsError fsErr volDBErr action =
     tryVolDB fsErr volDBErr action >>= either (throwError volDBErr) return
 
--- | Parses the 'FsPath' of each filename and zips them together.
--- If any 'FsPath' does not parse, we return an error.
+-- | Parses the 'FileId' of each 'FsPath' and zips them together.
+-- If any parsing does not parse, we abort with an error.
```

Comment:
`If any parsing does not parse, we abort with an error.` -> `When parsing fails, we abort with the corresponding parse error.`